_data/buildx/docker_buildx.yaml

```diff
@@ -7,6 +7,7 @@ cname:
     - docker buildx bake
     - docker buildx build
     - docker buildx create
+    - docker buildx debug-shell
```

Comment:
Hum yeah I don't think we should have this command here. Let me check.

Reply:
Actually it's fine, we also display experimental cmds and flags such as `--invoke`:

![image](https://github.com/docker/docs/assets/1951866/f7812d26-9542-4f74-83d5-44dbf9e1b78f)

Maybe we could mark this flag as experimental and display the label instead of adding this to the description.